packages/ariakit/src/suggestionMenu/SuggestionMenuLoader.tsx

```diff
@@ -6,13 +6,20 @@ export const SuggestionMenuLoader = forwardRef<
   HTMLDivElement,
   ComponentProps["SuggestionMenu"]["Loader"]
 >((props, ref) => {
-  const { className, children, ...rest } = props;
+  const { className, ...rest } = props;
 
   assertEmpty(rest);
 
   return (
     <div className={className} ref={ref}>
-      {children}
+      <svg
```

Comment:
did you copy this from anywhere? in that case always good to leave the source url as a comment